apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -72,6 +72,45 @@ type CleanupCronJobConfig struct {
 	Schedule string `json:"schedule,omitempty"`
 }
 
+type RegistryConfig struct {
+	// A registry where backup images are stored. Images are stored
+	// in {path}/${DEVWORKSPACE_NAMESPACE}/${DEVWORKSPACE_NAME}:latest
+	// +kubebuilder:validation:Required
+	Path string `json:"path,omitempty"`
+	// AuthSecret is the name of a Kubernetes secret of
+	// type kubernetes.io/dockerconfigjson.
+	// The secret is expected to be in the same namespace the workspace is running in.
+	// If secret is not found in the workspace namespace, the operator will look for the secret
+	// in the namespace where the operator is running in.
+	// as the DevWorkspaceOperatorCongfig.
+	// +kubebuilder:validation:Optional
+	AuthSecret string `json:"authSecret,omitempty"`
```

Comment:
@dkwon17 
We can add requirements for all secrets to have `controller.devfile.io/watch-secret=true`, in this case no need to use noncachingclient

Reply:
+1 to do that,

@Allda could we add this line in the description (line 86)?

```
The secret must have the controller.devfile.io/watch-secret=true label set.
```

and use the regular client in the `handleRegistryAuthSecret` function?